backend/python/migrations/versions/3f6fff20884c_initial_migration.py

```diff
@@ -0,0 +1,523 @@
+            nullable=False,
+        ),
+        sa.Column("translator_id", sa.Integer(), nullable=True),
+        sa.Column("reviewer_id", sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["reviewer_id"],
+            ["users.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["story_id"],
+            ["stories.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["translator_id"],
+            ["users.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_story_translations_reviewer_id"),
+        "story_translations",
+        ["reviewer_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_story_translations_translator_id"),
+        "story_translations",
+        ["translator_id"],
+        unique=False,
+    )
+
+    story_translation_contents = op.create_table(
+        "story_translation_contents",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("story_translation_id", sa.Integer(), nullable=False),
+        sa.Column("line_index", sa.Integer(), nullable=False),
+        sa.Column(
+            "status",
+            sa.Enum(
+                "name",
+                "DEFAULT",
+                "ACTION_REQUIRED",
+                "APPROVED",
+                name="storytranslationcontentstatus",
+            ),
+            server_default="DEFAULT",
+            nullable=False,
+        ),
+        sa.Column("translation_content", mysql.LONGTEXT(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["story_translation_id"],
+            ["story_translations.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_story_translation_contents_story_translation_id"),
+        "story_translation_contents",
+        ["story_translation_id"],
+        unique=False,
+    )
+    comments = op.create_table(
+        "comments",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("story_translation_content_id", sa.Integer(), nullable=False),
+        sa.Column("user_id", sa.Integer(), nullable=False),
+        sa.Column("comment_index", sa.Integer(), nullable=False),
+        sa.Column("time", mysql.DATETIME(), nullable=False),
+        sa.Column("resolved", sa.Boolean(), nullable=False),
+        sa.Column("content", mysql.LONGTEXT(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["story_translation_content_id"],
+            ["story_translation_contents.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["user_id"],
+            ["users.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_comments_story_translation_content_id"),
+        "comments",
+        ["story_translation_content_id"],
+        unique=False,
+    )
+    ### end Alembic commands ###
+
+    # seed database
+    op.bulk_insert(
```

Comment:
![image](https://user-images.githubusercontent.com/15113249/131426748-284723a9-72e9-4bdb-aed7-207e539b0b6d.png)
woops, didnt realize seeding would be included in this pr, closing #81. One modification I'll ask for: can you copy the descriptions and youtube links for stories from #81?

Reply:
Removed seeding; adding to this PR instead: https://github.com/uwblueprint/planet-read/pull/83